### DISCUSSION TITLE
How to avoid full path names for volume mounts in Docker

### DISCUSSION THREAD

**Discussion by jayguidos** [Camelcade/Perl5-IDEA#2478] [General]
Hi, we are using the Docker integration feature of the plugin.  You provide a way to specify docker parameters, and it works ok, but we want to check in the configuration.  The problem is there is no way to avoid full paths:

-e
TNS_ADMIN=/etc/oracle
--user=2006
-v
/home/jay.guidos/projects/perl/perl-ops/.local/appl/bids/etc:/appl/bids/etc:ro

I can't really check in this because of the hard coded path that only works for my machine.  I would like to specify paths that are reusable, like maybe $PRJOECT_DIR/.local/appl/bids/etc:/appl/bids/etc:ro but these do not seem work.  How can I get around this problem?

**Comment by hurricup** [comment]
I can add the possibility to use these project macros. For now you can probably use the environment variable. Anyway, this is a nice feature request.
